docs/_assets/codepen.js

```diff
@@ -102,6 +102,15 @@ CodeBlockCodePen.prototype.clickHandler = function(form, pre) {
 
   return function() {
 
+    // Track codepen button clicks
+    if (typeof ga !== 'undefined') {
+      ga('send', {
+        hitType: 'event',
+        eventCategory: 'codepen',
+        eventAction: 'click'
```

Comment:
Because the codepen button is only aware of what's inside the `<pre>` block I don't have a good way of knowing it's the "Raised Colored Button" codepen  button...